docs/source/DeveloperGuide/AddingNewComponents/DevelopingCommands/index.rst

```diff
@@ -4,7 +4,7 @@ Developing Commands
 
 .. warning::
     This instructions here demonstrate how to support both the legacy (Python 2) and future (Python3) installations of DIRAC.
-    If only having Python 3 support is acceptable, the requirement for scripts to be in the the *scripts* directory of their parent system will be removed and the only requirement will be for the function to be decorated with the ``@DIRACScript()`` decorator.
+    If only having Python 3 support is acceptable, the requirement for scripts to be in the the *scripts* directory of their parent system will be removed and the only requirement will be for the function to be decorated with the ``@Script()`` decorator.
```

Comment:
```suggestion
    If only having Python 3 support is acceptable, the requirement for scripts to be in the the *scripts* directory of their parent system will be removed and the only requirement will be for the function to be decorated with the ``@DIRACScript()`` decorator (or ``@Script`` if the DIRACScript class is imported as Script).
```
@fstagni, does such a description will be OK?

Reply:
yes, perfect.